### ISSUE TITLE
mat-checkbox content is not announced as label on IE11

### ISSUE THREAD

**Issue by architshukla** [angular/components#10157]
#### Bug, feature request, or proposal:
Bug verified on Edge and IE11 using Narrator and NVDA.

#### What is the expected behavior?
The content of `mat-checkbox` should be read as the label for the checkbox.

#### What is the current behavior?
Label is not read by screen readers on Edge and IE11. An explicit `aria-label` or `aria-labelledby` has to be used as a workaround.

#### What are the steps to reproduce?
1. Go to any of these links on Edge or IE
    - Angular Material Docs: https://material.angular.io/components/checkbox/overview
    - StackBlitz: https://angular-material2-issue-wxmigr.stackblitz.io
2. Tab to the checkbox
3. The screen reader reads "checkbox, not checked" but not the label associated to the checkbox

#### What is the use-case or motivation for changing an existing behavior?
- Screen reader users rely on the context provided by the checkbox label
- Consistent behavior of controls across browsers

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 5.2.6
Angular Material 5.2.2
Windows 10
TypeScript 2.4.2
Browsers - Edge and IE

**Comment by jelbourn** [comment]
@tinayuangao It would be worth checking if putting `role="presentation"` on the div between the `<label>` and the `<input>` resolves this.

**Comment by jelbourn** [comment]
I just tested this with IE11 and NVDA and the label works as expected in the latest version.
